addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -69,149 +64,136 @@
+            <h3>Focus</h3>
+            <p>Building connections</p>
         </div>
-
-        <div class="o_grid_item col-lg-6 g-col-lg-6 g-height-10 o_grid_item_image text-center" data-name="Block"
-            style="grid-area: 1 / 7 / 11 / 13; z-index: 4;">
-            <img src="/web/image/website.s_masonry_block_default_image_2" class="img img-fluid mx-auto" alt=""/>
+        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-2 o_cc o_cc3 o_colored_level justify-content-start rounded-3" data-name="Block" style="z-index: 1; grid-area: 2 / 1 / 5 / 4; --grid-item-padding-y: 20px; --grid-item-padding-x: 20px; background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-2) 100%);">
+            <h3>Innovation</h3>
+            <p>How ideas come to life</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-2 text-center o_cc o_cc2" data-name="Block"
-            style="grid-area: 11 / 7 / 14 / 10; z-index: 5;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-2 text-center o_cc o_cc2 o_colored_level rounded-3" data-name="Block" style="z-index: 5; --grid-item-padding-y: 20px; --grid-item-padding-x: 20px; grid-area: 9 / 7 / 12 / 10;">
+            <h3><span class="display-1">22&#37;</span></h3>
+            <p>Reaching new heights</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-2 text-center o_cc o_cc4" data-name="Block"
-            style="grid-area: 11 / 10 / 14 / 13; z-index: 6;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-2 text-center o_cc o_cc4 o_colored_level o_draggable rounded-3" data-name="Block" style="z-index: 6; --grid-item-padding-y: 20px; --grid-item-padding-x: 20px; grid-area: 9 / 10 / 12 / 13; background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-1) 100%);">
+            <h3><span class="display-1">+12​</span><br/></h3>
+            <p>Making the difference</p>
         </div>
     </div>
 </template>
 
 <template id="s_masonry_block_texts_image_texts_template" groups="base.group_user">
-    <div class="row o_grid_mode" data-row-count="10">
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc3" data-name="Block"
-            style="grid-area: 1 / 1 / 6 / 4; z-index: 1;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+    <div class="row o_grid_mode" data-row-count="9" style="gap: 16px;">
+        <div class="o_grid_item g-col-lg-3 text-center o_cc o_cc3 o_colored_level g-height-5 col-lg-3 rounded-4" data-name="Block" style="grid-area: 1 / 1 / 6 / 4; z-index: 1; background-image: linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-2) 100%);">
```

Comment:
Same vertical issue, the "centered" button in the WB is not selected.
Add `justify-content-center`

Same for the other blocks of this template ;)